## W0703 (broad-except)

### :x: Problematic code:

```python
try:
    1 / 0
except Exception:
    pass
```

### :heavy_check_mark: Correct code:

```python
try:
    1 / 0
except ZeroDivisionError:
    pass
```

### Rationale:

Used when an except catches a too general exception, possibly burying
unrelated errors.

### Related resources:

- [Issue Tracker](https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22broad-except%22+OR+%22W0703%22)
